Dockerfile

```diff
@@ -15,21 +15,21 @@ RUN go mod download && \
 
 ARG QASE_TEST_RUN_ID
 ARG TERRAFORM_VERSION
-ARG EXTERNAL_ENCODED_VPN
-ARG VPN_ENCODED_LOGIN
 ARG RKE_PROVIDER_VERSION
 ARG RANCHER2_PROVIDER_VERSION
 ARG LOCALS_PROVIDER_VERSION
 ARG AWS_PROVIDER_VERSION
+ARG KUBERNETES_PROVIDER_VERSION
```

Comment:
I'm assuming this is needed for just Harvester, right? To your point in the PR description, we have planned to consolidate the provider versions at a later point in time, it's not high priority atm.